datarepo-clienttests/src/main/java/scripts/testscripts/BillingProfileRoleAccessTest.java

```diff
@@ -0,0 +1,81 @@
+package scripts.testscripts;
+
+import bio.terra.datarepo.model.BillingProfileModel;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.config.TestUserSpecification;
+import scripts.testscripts.baseclasses.BillingProfileUsers;
+import scripts.utils.tdrwrapper.DataRepoWrap;
+
+public class BillingProfileRoleAccessTest extends BillingProfileUsers {
+  private static final Logger logger = LoggerFactory.getLogger(BillingProfileRoleAccessTest.class);
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public BillingProfileRoleAccessTest() {
+    super();
+  }
+
+  private BillingProfileModel profile;
+  private String stewardsEmail;
+
+  public void setup(List<TestUserSpecification> testUsers) throws Exception {
+    super.setup(testUsers);
+  }
+
+  public void setParameters(List<String> parameters) throws Exception {
+    if (parameters == null || parameters.size() == 0) {
+      throw new IllegalArgumentException(
+          "Must provide a number of files to load in the parameters list");
+    } else {
+      stewardsEmail = parameters.get(0);
+    }
+  }
+
+  public void userJourney(TestUserSpecification testUser) throws Exception {
+    DataRepoWrap ownerUser1Api = DataRepoWrap.wrapFactory(ownerUser1, server); // dumbledore
+    DataRepoWrap ownerUser2Api = DataRepoWrap.wrapFactory(ownerUser2, server); // voldemort
+    DataRepoWrap userUserApi = DataRepoWrap.wrapFactory(userUser, server); // mcgonagall
+
+    try {
+      // TODO: This test will stop working when DR-1475 is complete. At
+      //  that point we will need to switch to using a test billing account that we can manipulate
+      //  to add specific users with the ability to create profiles.
```

Comment:
I think the DR-1475 PR was just merged. Can this switch happen now?

Reply:
I talked with @fboulnois and it turns out that dumbledore and voldemort both have create access to our usual billing account, so this is ok. I did have to rearrange the rules in owner hand off tests so that only the authorized users tried to create billing profile.